verifier/verifier.py

```diff
@@ -24,252 +24,255 @@
-
-        # Initialize times for assets processing profiling
-        start = time.clock()
-        start_user = time.time()
-
-        # Instantiate VideoAssetProcessor class
-        asset_processor = VideoAssetProcessor(source,
-                                              pre_verified_renditions,
-                                              metrics_list,
-                                              do_profiling,
-                                              max_samples,
-                                              features)
-
-        # Record time for class initialization
-        initialize_time = time.clock() - start
-        initialize_time_user = time.time() - start_user
-
-        # Register times for asset processing
-        start = time.clock()
-        start_user = time.time()
-
-        # Assemble output dataframe with processed metrics
-        metrics_df, pixels_df, dimensions_df = asset_processor.process()
-
-        # Record time for processing of assets metrics
-        process_time = time.clock() - start
-        process_time_user = time.time() - start_user
-
-        # Normalize input data using the associated scaler
-        x_renditions = np.asarray(metrics_df)
-        x_renditions = loaded_scaler.transform(x_renditions)
-       
-        # Make predictions for given data
-        start = time.clock()
-        y_pred = loaded_model.decision_function(x_renditions)
-        prediction_time = time.clock() - start
-
-        # Add predictions to rendition dictionary
-        i = 0
-        for _, rendition in enumerate(renditions):
-            if rendition['video_available']:
-                rendition.pop('path', None)
-                rendition['tamper'] = np.round(y_pred[i], 6)
-                # Append the post-verification of resolution and pixel count
-                if 'pixels' in rendition:
-                    rendition['pixels_post_verification'] = float(rendition['pixels']) / pixels_df[i]
-                if 'resolution' in rendition:
-                    rendition['resolution']['height_post_verification'] = float(rendition['resolution']['height']) / int(dimensions_df[i].split(':')[0])
-                    rendition['resolution']['width_post_verification'] = float(rendition['resolution']['width']) / int(dimensions_df[i].split(':')[1])
-                i += 1
-
-        if do_profiling:
-            print('Features used:', features)
-            print('Total CPU time:', time.clock() - total_start)
-            print('Total user time:', time.time() - total_start_user)
-            print('Initialization CPU time:', initialize_time)
-            print('Initialization user time:', initialize_time_user)
-
-            print('Process CPU time:', process_time)
-            print('Process user time:', process_time_user)
-            print('Prediction CPU time:', prediction_time)
-
-    return renditions
+	"""
+	Function to verify that rendition conditions and specifications
```

Comment:
Looks like most of the changes in this file are the result of a change in indentation. Could we use the same indentation that is currently used in the file? This would be helpful for differentiating between formatting vs. functional changes.

Reply:
It's best to use consistent indentation across the whole project. Switched IDE to spaces and updated the file.